docs/BUILDING_MACOS.md

```diff
@@ -43,10 +43,9 @@ cd ~/binutils-tmp
 
 Get and extract binutils source
 ```bash
-wget https://ftp.gnu.org/gnu/binutils/binutils-2.35.tar.bz2
+curl -O https://ftp.gnu.org/gnu/binutils/binutils-2.35.tar.bz2
```

Comment:
Why use curl over `wget`? :thinking:

Reply:
tbh mostly because `curl` is pre-installed on mac but `wget` is not. I think `curl`'s output looks slightly better too though